src/prompts/prompt.js

```diff
@@ -58,6 +68,13 @@ function prompt (options, done) {
   function focusDialog () {
     dom.input.focus();
   }
+  
+  function rootClick (e) {
+    var classlist = e.target.classList.value;
```

Comment:
`classlist` is too generic. Which element is this that we are referencing? Also, follow the camelCase naming convention.